tests/helpers.ts

```diff
@@ -128,3 +140,31 @@ export async function verify(fixtureName: string) {
   await build(project);
   await emberTest(project);
 }
+
+export async function fastVerify(fixtureName: string) {
+  let project = await addonFrom(fixtureName);
+
+  await migrate(project);
+
+  const result = await snapshotFrom(project.baseDir);
+
+  const fixturePath = join(__dirname, 'samples', fixtureName);
+
+  const outputPath = join(fixturePath, 'output.json');
+
+  if (fs.existsSync(outputPath)) {
+    const prevResult = JSON.parse(fs.readFileSync(outputPath, 'utf8'));
```

Comment:
Why so many sync fs operations? Why not async?

Reply:
it's not meaningful for speed in this context, overall test per addon took 120s, and here there is no reason to add async logic

Reply:
Ok, sounds good 👍

Reply:
but, I can add it needed, don't have strong position here

Reply:
All good. I'm not sure i do either. I've developed a habit of using fs/promises everywhere though. So the introduction of sync fs code is new to this project 🤷